src/views/components/cart-panel.jsx

```diff
@@ -19,12 +19,20 @@ export default class CartPanel extends React.Component {
         if(regex.test(e.target.value)) {
             try {
                 let result = readAddress(e.target.value);
-                assert.strictEqual(result.type, 'item');
+                assert(result.type === 'item' || result.type === 'model');
```

Comment:
This check is not needed. If the result.type is not item or model `readAddress` will throw an error anyways.